CI.rst

```diff
@@ -59,7 +59,7 @@ Container Registry used as cache
 We are using GitHub Container Registry to store the results of the ``Build Images``
 workflow which is used in the ``Tests`` workflow.
 
-Currently in main version of Airflow we run tests in 4 different versions of Python (3.7, 3.8, 3.9, 3.10)
+Currently in main version of Airflow we run tests in 4 different versions of Python (3.7, 3.8, 3.9, 3.10, 3.11)
```

Comment:
```suggestion
Currently in main version of Airflow we run tests in 5 different versions of Python (3.7, 3.8, 3.9, 3.10, 3.11)
```

Reply:
Right. But I prefer to change it to "In those versions of Python" :D to avoid future, similar problems